apps/spanishwotd/spanish_wotd.star

```diff
@@ -0,0 +1,105 @@
+"""
+Applet: Spanish WoD
+Summary: Word of the day in Spanish
+Description: Displays the spanish word of the day including definition and translation.
+Author: logancornelius
+"""
+
+load("cache.star", "cache")
+load("encoding/json.star", "json")
+load("html.star", "html")
+load("http.star", "http")
+load("render.star", "render")
+
+DEFAULT_WHO = "world"
+
+SPANISH_DICTIONARY_URL_1 = "https://www.dictionaryapi.com/api/v3/references/spanish/json/"
+SPANISH_DICTIONARY_URL_2 = "?key=d5ccb112-d698-45d6-ad07-c17a7b829b8d"
```

Comment:
You may want to generate a new key, add it as a `secret` and then deprecate this one. Your call, but this one is now effectively public information.